### PR TITLE
Multiple updates

### PR DESCRIPTION
**Pull request by OgreTransporter**
I've updated the german translation file (a95b373).

The MSVC has printed some simple compiler warnings regarding typecasting that are fixed by 464398f.

I've made big changes on the CMake scripts (4aa8b10). Normally under Windows the Qt SDK is not distributed with static libraries, but only with dynamic DLLs. Therefore dynamic linking is now default. However, the NPACKED_FORCE_STATIC_QT switch can be used to force static linking. Instead of completely setting each program individually, all global settings are now set in the CMakeLists.txt in the main directory. There you can also select the projects to be built via switches (NPACKD_BUILD_TESTS, NPACKED_BUILD_CLU, NPACKED_BUILD_NPACKDCL and NPACKED_BUILD_NPACKDG). When installing the dynamically linked projects, the DLLs are no longer copied manually, but by the Qt program windeployqt. The program copies all dependencies and plugins automatically.

### CODE REVIEW

File: npackdcl/src/app.cpp [hunk 1]
```diff
@@ -246,7 +246,7 @@ int App::process()
             build(job);
         } else {
             job->setErrorMessage(QStringLiteral("Wrong command: ") + cmd +
-                    QStringLiteral(". Try \"ncl help\""));
+                    QStringLiteral(". Try \"npackdcl help\""));
```
Comment (tim-lebedkov) [inline npackdcl/src/app.cpp]: "ncl" is actually the *new* name.

Comment (OgreTransporter) [reply npackdcl/src/app.cpp]: Fixed in 0c3714d

**Comment by tim-lebedkov** [comment]
I cherry-picked the 2 changes from this pull request: "update german translation" and "Fix compiler warnings". I will close this pull request. Please create a new pull request with CMake changes only where everything builds.

**Comment by OgreTransporter** [comment]
New request #51

**Pull request closed without merging**